### ISSUE TITLE
nit: doc about `stash`

### ISSUE THREAD

**Issue by fzyzcjy** [Nukesor/pueue#589]
Hi thanks for the tool! I found stash is helpful, but it seems the wiki does not mention it (I found it by looking at `pueue --help`).

**Comment by Nukesor** [comment]
The wiki is free to be edited by all :)
Feel free to add anything you find missing!

**Comment by fzyzcjy** [comment]
Looks great! But I am not very sure about the semantics of `stash`, so maybe I am not able to edit it accurately.

**Comment by Nukesor** [comment]
Fair point. The wiki needs some love anyhow and I'm currently working on an `mdbook` for another project as well.

Might migrate the wiki over to that at some point.

**Comment by Nukesor** [comment]
Added a section about `stash`:
https://github.com/Nukesor/pueue/wiki/Get-started#stashed-tasks
